README.md

```diff
@@ -74,8 +74,8 @@ obj.link(Link { from: "main", to: "printf", at: 29 })?;
 obj.link(Link { from: "main", to: "deadbeef", at: 10 })?;
 obj.link(Link { from: "deadbeef", to: "DEADBEEF", at: 7 })?;
 
-// Finally, we write which object file we desire
-obj.write::<Elf>(file)?;
```

Comment:
Not going to lie, this was one of my favorite features and sad it might go away.

I’m not sure how often the use case was where the flexibility was required to be lazy about final binary target, but I nevertheless thought it was awesome and unique.